### ISSUE TITLE
Monarch Gene Mappings not found

### ISSUE THREAD

**Issue by anitacaron** [monarch-initiative/monarch-mapping-commons#37]
I'm running `make mappings` to get the mappings, but I'm getting an error when downloading the `gene_mappings.tsv`. The folder `latest` is empty in the URL https://data.monarchinitiative.org/monarch-gene-mapping/index.html

```bash
mkdir -p mappings tmp
wget http://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.tsv -O tmp/gene_mappings.sssom.tsv
--2023-11-15 11:24:57--  http://data.monarchinitiative.org/monarch-gene-mapping/latest/gene_mappings.tsv
Resolving data.monarchinitiative.org (data.monarchinitiative.org)... 35.208.191.193
Connecting to data.monarchinitiative.org (data.monarchinitiative.org)|35.208.191.193|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-11-15 11:24:57 ERROR 404: Not Found.

make[1]: *** [mappings/gene_mappings.sssom.tsv] Error 8
```

This is needed for #29

**Comment by glass-ships** [comment]
we ran the gene-mapping generation overnight and didn't realize until just now that the Jenkins job doesn't run any indexing afterwards. the new gene mapping file is in the `data-public` bucket, however it won't be available at that particular url until the indexer gets ran. 
i'll look at migrating that code here in a future PR, we just wanted to fix the more pressing issues from #33 first

**Comment by glass-ships** [comment]
> so next step is fixing the url in this repo

i actually just did that in the latest commit in the `sssom-mappings` branch of this repo
